### DISCUSSION TITLE
Advice

### DISCUSSION THREAD

**Discussion by SatoshiNakamotogmx** [metaspartan/denarius#302] [Q&A]
I was hijacked in 2013 When my future wife health took a turn I had to focus on other things how do I get help to fix this issue with the Cyber stalker's?

**Comment by itsN1X** [comment]
What's the issue?

**Reply by SatoshiNakamotogmx** [reply]
I told you in 2013 my P2P was hijacked .  I want to make sure they don't hijack me again.  Sincerely, satoshi

**Reply by metaspartan** [reply]
> I told you in 2013 my P2P was hijacked . I want to make sure they don't hijack me again. Sincerely, satoshi

Are you referring to your GMX email account?

lol, I am still unsure what I am looking at here.

Simple answer: U2F or 2FA with physical security key

**Reply by SatoshiNakamotogmx** [reply]
😉👍

**Reply by SatoshiNakamotogmx** [reply]
See you soon

**Reply by metaspartan** [reply]
> See you soon

See who soon? Can you please explain why this thread is of any relevance to Denarius and why it should not be deleted?

**Reply by SatoshiNakamotogmx** [reply]
This Thread was a question from me Satoshi Nakamoto asking advice to make sure I don't get hijacked again like I did in 2013 and yeah you can delete the thread thank you.  What I mean is you'll see me coming back and you'll know it's me

**Reply by itsN1X** [reply]
⏿

**Reply by minecoins247365** [reply]
craig wright is more satoshi than this dude lol

**Reply by buzzkillb** [reply]
I just want to be in the screenshot for these breaking Denarius matters.

**Reply by SatoshiNakamotogmx** [reply]
Sorry you feel that way.

**Reply by SatoshiNakamotogmx** [reply]
> craig wright is more satoshi than this dude lol

 Really going to say that to me?   Sorry you feel that way.  I was giving a project to get working and did what I could for Nick szabo and harold.  Thank you for being so kind..  my paperwork say jan 3 at 5:49 am.  They said they would take care of all of the writing.  Then I was hijacked and had to jack it back and tell you I wasn't doren I couldn't let him go through that drama I felt bad .  You have a great day.  Sincerely the 44 year old from CA.  Satoshi Nakamoto was the name they gave me .  The same night I did the coding . 
My girlfriend aka wife was dieing and I had to walk away to take care of her she died in 2013 I took some time to find myself.

**Reply by metaspartan** [reply]
> > craig wright is more satoshi than this dude lol
> 
> Really going to say that to me? Sorry you feel that way. I was giving a project to get working and did what I could for Nick szabo and harold. Thank you for being so kind.. my paperwork say jan 3 at 5:49 am. They said they would take care of all of the writing. Then I was hijacked and had to jack it back and tell you I wasn't doren I couldn't let him go through that drama I felt bad . You have a great day. Sincerely the 44 year old from CA. Satoshi Nakamoto was the name they gave me . The same night I did the coding .
> My girlfriend aka wife was dieing and I had to walk away to take care of her she died in 2013 I took some time to find myself.

I would say you sound more like Satoshi than CW lmao

If you are, life happens, and welcome back, the world has changed a lot since 2013 lol and Blockstream hijacked your project lol

**Reply by buzzkillb** [reply]
Denarius is a great place to say what's on your mind about cryptocurrencies. Are you thinking of helping out? I hope you are back, that would be pretty game changing.

**Reply by SatoshiNakamotogmx** [reply]
> > > craig wright is more satoshi than this dude lol
> > 
> > Really going to say that to me? Sorry you feel that way. I was giving a project to get working and did what I could for Nick szabo and harold. Thank you for being so kind.. my paperwork say jan 3 at 5:49 am. They said they would take care of all of the writing. Then I was hijacked and had to jack it back and tell you I wasn't doren I couldn't let him go through that drama I felt bad . You have a great day. Sincerely the 44 year old from CA. Satoshi Nakamoto was the name they gave me . The same night I did the coding .
> > My girlfriend aka wife was dieing and I had to walk away to take care of her she died in 2013 I took some time to find myself.
> 
> I would say you sound more like Satoshi than CW lmao
> 
> If you are, life happens, and welcome back, the world has changed a lot since 2013 lol and Blockstream hijacked your project lol

Thank you for everything and thanks for your patience.  Hope you all have a wonderful day.  Thank you again.  Sorry I left I had to care for the woman I love hope you can forgive me for that but my love needed me more at the time it was a no brainer..

**Reply by metaspartan** [reply]
> > > > craig wright is more satoshi than this dude lol
> > > 
> > > 
> > > Really going to say that to me? Sorry you feel that way. I was giving a project to get working and did what I could for Nick szabo and harold. Thank you for being so kind.. my paperwork say jan 3 at 5:49 am. They said they would take care of all of the writing. Then I was hijacked and had to jack it back and tell you I wasn't doren I couldn't let him go through that drama I felt bad . You have a great day. Sincerely the 44 year old from CA. Satoshi Nakamoto was the name they gave me . The same night I did the coding .
> > > My girlfriend aka wife was dieing and I had to walk away to take care of her she died in 2013 I took some time to find myself.
> > 
> > 
> > I would say you sound more like Satoshi than CW lmao
> > If you are, life happens, and welcome back, the world has changed a lot since 2013 lol and Blockstream hijacked your project lol
> 
> Thank you for everything and thanks for your patience. Hope you all have a wonderful day. Thank you again. Sorry I left I had to care for the woman I love hope you can forgive me for that but my love needed me more at the time it was a no brainer..


We are always welcoming new and old developers to Denarius to assist me with development, it is a now an almost 4 year old project. What would you want most changed in Bitcoin now?

**Reply by SatoshiNakamotogmx** [reply]
> Denarius is a great place to say what's on your mind about cryptocurrencies. Are you thinking of helping out? I hope you are back, that would be pretty game changing.

Yes I am going to help anyway I can.  I just don't want to make a fuss that's why I came and asked advice.  Kinda quietly work back in the mix.   After all we got everything we need.  LOL

**Comment by itsN1X** [comment]
Add more decimals for One.
